docs/spec/reactors/mempool/README.md

```diff
@@ -1,11 +0,0 @@
-# Mempool Specification
```

Comment:
why remove? was it moved somewhere?

Reply:
not moved. The only thing in this file is a ToC telling the reader about the 4 files in this directory. For consistency we should either have a README like this in every directory or none at all. The former option doesn't make much sense IMO

Reply:
maybe we can at least try to pattern all the reactor dirs the same way, so its always like `config.md, messages.md, state.md` or something